src/dex/aave-v2/tokens.ts

```diff
@@ -1,6 +1,6 @@
 import { aToken, Token } from '../../types';
 import { Network } from '../../constants';
-import { tokenlist } from '@bgd-labs/aave-address-book';
+import tokenlist from '@bgd-labs/aave-address-book/dist/tokenlist';
```

Comment:
I didn't manage to make typescript accept this.
On other projects ppl import `@bgd-labs/aave-address-book/tokenlist` just fine, but here i didn't figure out why it does not work. As on v2 for sure no more assets are added could make sense to just copy the list to the repo and drop the import.

Reply:
I don't mind copying the list to the repo, not worth spending time trying to make it work